### ISSUE TITLE
ExTldr: new client for tldr-pages

### ISSUE THREAD

**Issue by ivanhercaz** [tldr-pages/tldr#4032]
Dear friends of tldr-pages!
It is a pleasure to regards you again since a lot of time I have been inactive. I have been very busy, and I still busy, so my contributions to `tldr` suffered a drastic reduction. I will try to contribute as soon as possible with some commands I have in mind.

Between the stuff I am doing, I am learning Elixir for a project, but before to apply it I need to test my skills and improve them. I was curious about the existence of a Elixir client for tldr-pages and I found [TLDR Elixir Client](https://github.com/edgurgel/tldr_elixir_client), but it is archived and it isn't updated since 6 years ago. So I decided to contribute to tldr-pages with a Elixir client and...

[ExTldr](https://github.com/ivanhercaz/extldr) arrived! At this moment it is a very basic client, it just supports search a command using `extldr operating_system command`, but I will work in the multilingual support (ivanhercaz/extldr#1) as soon as possible. It just has one requirements, in addition to the one of download it: it is necessary to have Erlang installed (not Elixir).

Well, besides to announce you this new client, I would like to know if it could be added to the client list and what should we do with the current Elixir client in the list.

Again, it is a great pleasure to regard all of you. I hope everyone is nice and no one is affected of this horrible pandemic everyone is suffering around the world.

Huge hugs for everyone and good health!

**P.S.** Of course, I think it is not necessary to say but, it is more than welcome all suggestions or recommendations :heart: 
**P.S. (2)** v0.2.0 released, now ExTldr detect automatically the operating system of the user.

**Comment by sbrl** [comment]
Hey there, @ivanhercaz! Thanks so much for the new client! Of course we can add it to the client list - it sounds like you're going to be putting lots of effort into developing and maintaining it (and, indeed, you have already!).

There are 2 places if I recall correctly that you can add your client to:

 - The wiki (no pull request required; can't remember if the list is still there or not)
 - The README (would you like to open a PR?)

**Comment by zdroid** [comment]
Nice work. I agree with @sbrl's suggestions, you should definitely submit a PR to add it to the client's list. :smile:

**Comment by ivanhercaz** [comment]
Thank you very much for your comments, @sbrl and @zdroid!

> it sounds like you're going to be putting lots of effort into developing and maintaining it (and, indeed, you have already!)

Yes, I am going to try the best of me with ExTldr due two reasons: 1. I want to make a great contribution to the tldr-pages community, 2. I really want to learn Elixir and this is a great opportunity. Probably there will be lapsus of time in which I am not going to be able to develop it continuously, due I am finishing my master degree and have some works to finish, but I hope to accomplish with the client specification (ivanhercaz/extldr#6) before July.

> - The wiki (no pull request required; can't remember if the list is still there or not)
> - The README (would you like to open a PR?)

> you should definitely submit a PR to add it to the client's list. :smile:

Nice! I am going to add it in the wiki if the list still exists and I am going to open a PR including it in the README. What should we do with the current Elixir client of the list? I ask it because it is archive and seems a non active project.

One thing more: if tldr-pages team is agree, it would be a pleasure for me to transfer this repository to tldr-pages organization.

Again, thank you sbrl and zdroid for your comments!

**Comment by zdroid** [comment]
I am not the org owner (yet!) so I can't help about the transferring part. I guess it is okay to have such repo as part of this org if it is going to be actively maintained, but again, the decision is not on me.

**Comment by ivanhercaz** [comment]
Although it is closed, I would like to know what think the owners about transfering the ExTldr repository to tldr organization.

CC. @agnivade @mebeim @owenvoke @rprieto @sbrl

**Comment by owenvoke** [comment]
Personally I think that would be great to have it under the @tldr-pages org. 👍 I'll let the other maintainers chip in though.

**Comment by sbrl** [comment]
Hey, transferring would be great! I can certainly do that for you. You'll of course retain admin access over the repository.

I believe that you need to do initiate the transfer though by going tot he danger zone in the settings and transferring the repository  into the `tldr-pages` org. Then I can fiddle with the settings for the new repository and add you as an admin over it @ivanhercaz

**Comment by ivanhercaz** [comment]
I am really happy both of you, @owenvoke and @sbrl, think transferring would be great :heart: Both opinions encourage me a lot. I was waiting for more opinions of @tldr-pages owners, but if I am understanding everything, you (sbrl) are telling me to initiate the process, so... I am going to do it as soon as possible!

**Comment by ivanhercaz** [comment]
Ups! @sbrl, it seems it was not necessary by the part of the owner to accept the transfer. I think it is because I am member of tldr-pages.

>  I can fiddle with the settings for the new repository and add you as an admin over it @ivanhercaz

This is the only change it might be necessary, because I think I will need to make work the services I had integrated in the ExTldr when it was part of my personal repositories. I opened an issue to remember that it is necessary to perform some changes after this transfer (tldr-pages/extldr#13).
